contracts/Coordinator.sol

```diff
@@ -79,11 +82,18 @@ contract Coordinator is ICoordinator {
     function withdrawCollateralUnderNFT(uint256 amount, uint256 nftId) external override notImplementedYet {}
 
     function borrowUnderNFT(uint256 _nftId, uint256 _amount) external override {
```

Comment:
It worth adding a comment explaining the param
e.g.: _amount is OUSD or lvUSD?

Reply:
Yes will add in variable name

Reply:
Adding to to methods in PR. Everything else will get updated as we go implement